specs/command-line-user-experience.md

```diff
@@ -288,6 +288,14 @@ lxc image import centos-7-x86\_64.tar.gz --created-at=2014-12-10 --expires-at=20
 lxc image import debian-jessie\_amd64.tar.gz dakara:                                                                    | Import a debian LXD image in the lxc image store of remote host "dakara"
 lxc image alias create centos/7 \<hash\>                                                                                | Create an alias for centos/7 pointing to our centos 7 image
 
+**Example output (lxc image list)**
+
+    HASH            PUBLIC    NAME                                  DESCRIPTION                     ALIAS
+    -----------------------------------------------------------------------------------------------------
+    146246146827... yes       busybox-amd64                         -                               busybox-amd64
+    95830b5e4e04... yes       ubuntu-vivid-amd64-default-20150308   Ubuntu 15.04 (devel) x86 64bit  ubuntu/devel (3 more)
```

Comment:
On Mon, Mar 09, 2015 at 10:30:03AM -0700, St�phane Graber wrote:
> > @@ -288,6 +288,14 @@ lxc image import centos-7-x86\_64.tar.gz --created-at=2014-12-10 --expires-at=20
> >  lxc image import debian-jessie\_amd64.tar.gz dakara:                                                                    | Import a debian LXD image in the lxc image store of remote host "dakara"
> >  lxc image alias create centos/7 \<hash\>                                                                                | Create an alias for centos/7 pointing to our centos 7 image
> >  
> > +**Example output (lxc image list)**
> > +
> > +    HASH            PUBLIC    NAME                                  DESCRIPTION                     ALIAS
> > +    -----------------------------------------------------------------------------------------------------
> > +    146246146827... yes       busybox-amd64                         -                               busybox-amd64
> > +    95830b5e4e04... yes       ubuntu-vivid-amd64-default-20150308   Ubuntu 15.04 (devel) x86 64bit  ubuntu/devel (3 more)
> 
> If the name property seems too confusing, I'm happy to just drop that special casing and not have the name column in there at all.

No, I think it's useful. I'd just put alias first (maybe alias, hash,
public, description, name)?.

> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/347/files#r26056587